src/OrchardCore.Modules/OrchardCore.Lists/GraphQL/ListQueryObjectType.cs

```diff
@@ -35,15 +35,14 @@ public ListQueryObjectType(IStringLocalizer<ListQueryObjectType> S)
                     var session = serviceProvider.GetService<ISession>();
                     var accessor = serviceProvider.GetRequiredService<IDataLoaderContextAccessor>();
 
-                    var dataLoader = accessor.Context.GetOrAddCollectionBatchLoader<string, ContentItem>("ContainedPublishedContentItems", x => LoadPublishedContentItemsForListAsync(x, session));
+                    var dataLoader = accessor.Context.GetOrAddCollectionBatchLoader<string, ContentItem>("ContainedPublishedContentItems",
+                                                    x => LoadPublishedContentItemsForListAsync(x, session, g.GetArgument<int>("skip"), g.GetArgument<int>("first")));
```

Comment:
```suggestion
                    var dataLoader = accessor.Context.GetOrAddCollectionBatchLoader<string, ContentItem>("ContainedPublishedContentItems",
                       x => LoadPublishedContentItemsForListAsync(x, session, g.GetArgument<int>("skip"), g.GetArgument<int>("first")));
```

Reply:
Does "first" specify the amount of the fetched rows?

Reply:
Yes, in the context of the code, it defaults to 10